### PR TITLE
Added Image uploading functionality

### PR DESCRIPTION
**Pull request by greekavi**

### CODE REVIEW

File: client/components/Menu.js [hunk 1]
```diff
@@ -216,11 +209,52 @@ export default function MenuTable({ storeId }) {
...
+    const addOrEdit = (item, resetForm,uploadImageFileName) => {
+        if (item.id === "") {
+            console.log(item.category)
+            if(uploadImageFileName!=null){
+                uploadImage({
+                    variables: {
+                        uploadImageFile:uploadImageFileName
+                    }
+                }).then(result=>{
+                    console.log(Object.values(result)[0].uploadImage.generatedFileName)
```
Comment (eyeamkd) [inline client/components/Menu.js]: Please remove console logs or comment them if they're absolutely necessary

File: server/resolvers/ItemResolver.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 //import { Stream } from "stream";
 
-const {stream} = require("stream")
+// const {stream} = require("stream")
```
Comment (eyeamkd) [inline server/resolvers/ItemResolver.js]: Remove comments if not necessary

File: server/resolvers/ItemResolver.js [hunk 3]
```diff
@@ -9,29 +9,56 @@ const mongoose = require('mongoose');
...
+                });
+                readstream.on('end', async () => {
+                  let bufferBase64 = Buffer.concat(data);
+                  const img = bufferBase64.toString('base64');
+                  resolve(img);
+                });
+              });
+        //    const stream=bucket.openDownloadStreamByName(id)
+        //    let buffer
+        //    stream.on('data',(chunk)=>{
```
Comment (eyeamkd) [inline server/resolvers/ItemResolver.js]: Remove commented code if not necessary

**Review by eyeamkd — CHANGES_REQUESTED** [review]
Hey @greekavi , please check the comments that I've made and push the required changes

File: client/components/Menu.js (hunk 1 above)
Comment (greekavi) [reply client/components/Menu.js]: Issue resolved

File: server/resolvers/ItemResolver.js (hunk 2 above)
Comment (greekavi) [reply server/resolvers/ItemResolver.js]: issue resolved

File: server/resolvers/ItemResolver.js (hunk 3 above)
Comment (greekavi) [reply server/resolvers/ItemResolver.js]: issue resolved

**Pull request merged**